app/src/main/java/org/mozilla/focus/fragment/WebFragment.java

```diff
@@ -38,6 +38,7 @@
      */
     @Nullable
     public abstract String getInitialUrl();
+    public abstract void onCreateViewCalled();
```

Comment:
Add a comment explaining what this method is and why it's necessary (because `onCreateView` is final so others don't forget to call it).